code/modules/cargo/exports.dm

```diff
@@ -53,9 +53,14 @@ Credit dupes that require a lot of manual work shouldn't be removed, unless they
 			if(E.applies_to(thing, allowed_categories, apply_elastic))
 				sold = E.sell_object(thing, report, dry_run, allowed_categories , apply_elastic, profit_ratio)
 				report.exported_atoms += " [thing.name]"
+				sold = TRUE
```

Comment:
Is it intended to overwrite the previous assignment? If so, why is the output of `E.sell_object()` being assigned to `sold` at all?